config/routes.rb

```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  namespace :api do
+    namespace :v1 do
+      resources :orders, only: [:index]
```

Comment:
I went the route of adding the api version in the url. The logic behind this is that it is the most widely used and understood pattern and is less error prone with older browsers when compared to more technically sound solutions.

The alternative I debated with is not including the api version in the url at all and instead referencing the version within the HTTP header. Specifically creating a custom vendor media type (e.g. `application/vnd.orders.{version_number}+json`).

You would in theory access it as such: `http :3000/orders Accept:'application/vnd.orders.v1+json'`.

See [this](https://stackoverflow.com/questions/389169/best-practices-for-api-versioning) page on StackOverflow that talks a bit about this topic.